bin/bank/pycbc_brute_bank

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python
+import numpy, h5py, logging, argparse, numpy.random, sys
+import pycbc.waveform, pycbc.filter, pycbc.types, pycbc.psd, pycbc.fft, pycbc.pnutils
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--verbose', action='store_true')
+parser.add_argument('--output-file',
+    help='Output file name for template bank.')
+parser.add_argument('--input-file',
+    help='Bank to use as a starting point.')
+parser.add_argument('--params',
+    help='list of paramters to use', nargs='+')
+parser.add_argument('--min',
+    help='list of the minimum parameter values', nargs='+', type=float)
+parser.add_argument('--max',
+    help='list of the maximum parameter values', nargs='+', type=float)
+parser.add_argument('--approximant', 
+    help='The waveform approximant to place')
+parser.add_argument('--minimal-match', type=float)
+parser.add_argument('--buffer-length', type=float,
+    help='size of waveform buffer in seconds')
+parser.add_argument('--sample-rate', type=int,
+    help='sample rate in seconds')
+parser.add_argument('--low-frequency-cutoff', type=float)
+parser.add_argument('--enable-sigma-bound', action='store_true')
+parser.add_argument('--tau0-threshold', type=float)
+parser.add_argument('--permissive', action='store_true',
+    help='Allow waveform generator to fail.')
+parser.add_argument('--seed', type=int, default=0)
+parser.add_argument('--tolerance', type=float)
+parser.add_argument('--max-mtotal', type=float)
+pycbc.psd.insert_psd_option_group(parser)
+args = parser.parse_args()
+pycbc.init_logging(args.verbose)
+numpy.random.seed(args.seed)
+
+class Shrinker(object):
+    def __init__(self, data):
+        self.data = data 
+
+    def pop(self):
+        if len(self.data) == 0:
+            return None
+        l = self.data[-1]
+        self.data = self.data[:-1]
+        return l
+
+class TriangleBank(object):
```

Comment:
TriangleBank may not sound immediately obvious to everyone, can you add a brief docstring?